src/components/PopupBase/__tests__/PopupBase.spec.tsx

```diff
@@ -44,119 +50,120 @@ describe('<PopupBase> that adds basic anchor/popup functionality to rendered com
             );
 
             // trigger setPopupVisibility(true) through our dummy component
-            wrapper.find('button').simulate('click');
+            await user.click(screen.getAllByRole('button', { name: 'Toggle popup' })[0]);
 
-            expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('Popover')).toHaveLength(1);
-            expect(wrapper.find('Portal')).toHaveLength(1);
+            expect(view.container).toMatchSnapshot();
+            expect(screen.getByRole('group')).toBeInTheDocument();
         });
-        it('should support no popup content from renderer', () => {
-            wrapper = mount(
+
+        it('should support no popup content from renderer', async () => {
+            view.rerender(
                 <PopupBase anchorRenderer={anchorRendererMock} popupRenderer={() => null} />
             );
 
             // trigger setPopupVisibility(true) through our dummy component
-            wrapper.find('button').simulate('click');
+            await user.click(screen.getAllByRole('button')[1]);
 
-            expect(toJson(wrapper)).toMatchSnapshot();
-            expect(wrapper.find('Popover')).toHaveLength(0);
+            expect(view.container).toMatchSnapshot();
```

Comment:
same as above

Reply:
If i use baseElement it will render 
![image](https://github.com/textkernel/oneui/assets/104063210/3a1b1140-0265-4a03-ae0e-e02c892c949c)